.github/workflows/tests.yaml

```diff
@@ -203,3 +203,72 @@ jobs:
       continue-on-error: true
       timeout-minutes: 60
       if: failure() && runner.debug == '1'
+
+  e2e-tests:
+    needs: build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        profile: [ 's3c' ]
+        cloudserver_tag: [ '7.70.47' ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '16'
+          cache: yarn
+      - name: Install node dependencies
+        run: yarn install --ignore-engines --frozen-lockfile --network-concurrency 1
+      - name: Login to Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ github.token }}
+      - name: Setup services
+        env:
+          KAFKA_IMAGE: ghcr.io/${{ github.repository }}/ci-kafka:${{ github.sha }}
+          REDIS_IMAGE: redis:alpine
+          METADATA_IMAGE: ghcr.io/scality/metadata:7.70.45-standalone
+          CLOUDSERVER_IMAGE: ghcr.io/scality/cloudserver:${{ matrix.cloudserver_tag }}
+        run: |-
+          set -e -o pipefail;
+          docker compose --profile ${{ matrix.profile }} up -d --quiet-pull
+          bash ../../scripts/wait_for_local_port.bash 8000 120
```

Comment:
i think there is now a way to add "health check" in docker compose, which would allow checking that services are "up" (i.e. answers on port) and thus wait on it...